constants/src/main/java/org/jboss/pnc/enums/ErrorType.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2020 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.enums;
+
+public enum ErrorType {
```

Comment:
ValidationErrorType? + javadoc on class level

Reply:
I don't like Java style of long names. I'd like to keep it simple and generic. This enum could be used somewhere else.

Reply:
Now both values are for validation. Do you have any example of other values that could be put here now?

Reply:
The enum is a simple collection of error types. If someone wants to specify a type of error in their DTO, he can add them to the enum and use it. Right now, only milestone validation uses it, so now no other values are needed.

Reply:
Then why would ValidationResponse class have generic error type that can have value that has nothing to do with validation? If it was a generic response class then I would understand that, but it's specific validation response.

Reply:
Ok, I am convinced.